client/packages/invoices/src/OutboundShipment/DetailView/AddFromScannerButton.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+import {
+  useTranslation,
+  useToggle,
+  InvoiceNodeStatus,
+  useBarcodeScannerContext,
+  CircularProgress,
+  ScanIcon,
+  ScanResult,
+  ButtonWithIcon,
+  useNotification,
+} from '@openmsupply-client/common';
+import {
+  ItemRowFragment,
+  MasterListSearchModal,
+} from '@openmsupply-client/system';
+import { useOutbound } from '../api';
+
+export const AddFromScannerButtonComponent = ({
+  onAddItem,
+}: {
+  onAddItem: (item?: ItemRowFragment) => void;
+}) => {
+  const t = useTranslation('distribution');
+  const { status } = useOutbound.document.fields(['status']);
+  const isDisabled = status !== InvoiceNodeStatus.New;
```

Comment:
Hmm, wouldn't this use `isOutboundDisabled` ?

Reply:
I thought that it was only for new shipments, I guess we can still add to picked and allocated - it's just not the usual workflow. will update, ta.